astropy/utils/data_info.py

```diff
@@ -623,8 +624,9 @@ def slice_indices(self, col_slice, item, col_len):
 
         Parameters
         ----------
-        col_slice : Column or mixin
-            Sliced object
+        col_slice : `~astropy.table.Column` or class
```

Comment:
We actually have a glossary term "-like' that can be combined with anything! However, the glossary defines it as 
```
      Used to indicate on object of that type or that can instantiate the type.
      E.g. :class:`~astropy.units.Quantity`-like includes ``"2 * u.km"``
      because ``astropy.units.Quantity("2 * u.km")`` works.
```
so this would be a different meaning. I suggest adding either a glossary term for mixin, or just adding it to `docs/conf.py::numpydoc_xref_ignore`. I think the latter is better, as not even Python bothers to explain Mixins on the official docs.

Reply:
Really it is a column duck type. But let's put it in `ignore` for now!